sql/mysqld.cc

```diff
@@ -3810,9 +3810,10 @@ static int generate_server_uuid()
 
   delete thd;
 
-  strncpy(server_uuid, uuid.c_ptr(), UUID_LENGTH);
+  strncpy(server_uuid, uuid.c_ptr(), sizeof(server_uuid));
```

Comment:
This is asking for ```static_assert(sizeof(server_uuid) >= UUID_LENGTH)``` but I am not sure it's our job to add it if no other comments on the PR.